core/domain/stats_jobs_one_off.py

```diff
@@ -195,3 +196,248 @@ def reduce(key, stringified_values):
                     'all: %s sum:%s' % (
                         key, state_name, all_state_hit[state_name],
                         sum_state_hit[state_name]),)
+
+
+class AnswerModelInteractionConsistencyJob(jobs.BaseMapReduceJobManager):
```

Comment:
We should add comments to these jobs to say that they will be removed after release 2.5.x, right? (Not sure what x is.)